packaging/deb/control/prerm

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
```

Comment:
I think we would prefer the shebang to look like this: `#!/usr/bin/env sh`

https://unix.stackexchange.com/questions/12736/how-does-usr-bin-env-know-which-program-to-use/12751#12751

Reply:
Usually I'd do so as well. Official Debian repo packages however use `#!/bin/sh` when I checked last. However, both will work reliably. I'll then also align this with the `postinst` script.

Reply:
Adjusted on both maintainer scripts.